SPA Template Fix - Replace redundant array creation expression with array initializer

iantoalms opened the pull request:
Simple change to remove redundant new keyword in array creation expression.

Addresses #10762

Kahbazi committed:
Remove nameof for event name

BrennanConroy committed:
Merged PR 891: Update MessagePack dependency

Update MessagePack dependency

Related work items: #142

dotnet-bot committed:
Merge in 'release/2.1' changes

natemcmaster committed:
Merge changes from 'release/2.1'

dotnet-bot committed:
Merge in 'release/2.1' changes

dotnet-bot committed:
Merge in 'release/2.1' changes

dotnet-bot committed:
Merge in 'release/2.1' changes

dotnet-bot committed:
Merge in 'release/2.1' changes

dotnet-bot committed:
Merge in 'release/2.1' changes

dotnet-bot committed:
Merge in 'release/2.1' changes

dotnet-bot committed:
Merge in 'release/2.1' changes

dotnet-bot committed:
Merge in 'release/2.1' changes

natemcmaster committed:
Move updateMigrations.cmd back into its correct subfolder

isaacrlevin committed:
#10333 Template Updates

isaacrlevin committed:
test failure fixes

isaacrlevin committed:
updates to pass tests

isaacrlevin committed:
Merge pull request #1 from aspnet/master

merge from main

isaacrlevin committed:
Update WebApiTemplateTest.cs

isaacrlevin committed:
test updates

isaacrlevin committed:
Update WebApiTemplateTest.cs

pranavkm committed:
Port do not throw when a status code with a codefix appears multiple times in the method body (#10235)

Fixes https://github.com/aspnet/AspNetCore/issues/4480

isaacrlevin committed:
api updates

isaacrlevin committed:
fix to f# controller

iantoalms committed:
Replace redundant array creation expression with array initializer

iantoalms committed:
Replace redundant array creation expression with array initializer

dnfclas commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/aspnet/AspNetCore?pullRequest=10787) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/aspnet/AspNetCore?pullRequest=10787) before we can accept your contribution.<br/><br/>:x: iantoalms [sign now](https://cla.dotnetfoundation.org/aspnet/AspNetCore?pullRequest=10787)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/aspnet/AspNetCore?pullRequest=10787) it.</sub>

dougbu committed:
Unpin EF dependencies then grab the latest (#10682)

- grab the latest from Blazor, EF Core (thanx @ajcvickers) and AspNetCore-Tooling
- partially reverts 58c67727dfb3

isaacrlevin committed:
Merge pull request #2 from aspnet/master

merge

isaacrlevin committed:
remove cdn from templates

pranavkm committed:
Revert "Add an option to SignalR to guarantee ordering of send events… (#10807)

This reverts commit 4646c4a5e4412dbff6e40ade887792241f5a2c94.

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/aspnet/AspNetCore-Tooling build 20190603.6 (#10797)

dougbu committed:
[release/3.0-preview6] Unpin EF dependencies then grab the latest (#10805)

- #10050
- includes new preview 6 packages from Blazor, Extensions, EntityFrameWorkCore, and AspNetCore-Tooling
- update coherent parent chain to use EF Core instead of AspNetCore-Tooling again
- partially reverts 58c67727dfb3

dougbu committed:
Add Microsoft.Extensions.ApiDesription.Server project and package (#10669)

- #8242 2 of 2, # #4912
- add missing project reference provider
  - remove incorrect `$(IsImplementationProject)` settings in `dotnet-getdocument` and `GetDocumentInsider`
  - re-run .\eng\scripts\GenerateProjectList.ps1

jkotalik committed:
Save condition backcollection between runs (#10776)

davidfowl committed:
If libuv doesn't unwind gracefully log a warning (#10823)

- We added 3 critical logs to see if there was a resource leak happeing during the loop thread shutdown, turns out that during testing its very common for the thread to not end immediately because of the latency of thread pool dispatching (the latency between queuing and execution of the work item). As a result, we log a warning instead of an error.

isaacrlevin committed:
remove cdn tests

isaacrlevin committed:
Update _ValidationScriptsPartial.cshtml

huan086 committed:
Add no-store to OpenIdConnectHandler cache-control response headers (#10318). (#10741)

analogrelay committed:
Merge pull request #6517 from Kahbazi/LogEventName#5

Remove nameof for event name

isaacrlevin committed:
remove cookie consent from templates

SteveSandersonMS committed:
Blazor template fixes (#10770)

* Invert UseLocalDB condition in template

* Replace UseBlazorClientSideFiles with UseClientSideBlazorFiles

* Add NotFoundContent to client-side templates, and make all consistent

* In client-side templates, remove references to MyGet feeds as they are no longer updated

* Workaround for lack of conditionals in .razor: Two separate versions of MainLayout.razor

* Workaround for lack of conditionals in .razor: Four variants of LoginDisplay.*.razor

* Fix launchSettings for Windows auth

natemcmaster committed:
Merge branch 'release/3.0-preview6'

dotnet-maestro[bot] committed:
Update dependencies from aspnet/Blazor (#10816)

- Microsoft.AspNetCore.Blazor.Mono - 0.10.0-preview6.19303.4

natemcmaster committed:
Update to latest dependencies

davidfowl committed:
Collect dumps for tests that seem to be flaky (#10830)

isaacrlevin committed:
add removed section

ryanbrandenburg commented:
Seems reasonably since it causes a warning in VSMac out of the box. I'll re-run the tests since they failed for reasons unrelated to this PR.

smitpatel committed:
Target netstandard2.1 in HealthChecks.EFCore

pranavkm committed:
Add Mvc.Api.Analyzers to 2.2.6 patch (#10621)

isaacrlevin committed:
merge (#3)

merge 

Co-authored-by: Kahbazi <a.kahbazi@gmail.com>
Co-authored-by: Pranav K <prkrishn@hotmail.com>
Co-authored-by: null <dotnet-maestro[bot]@users.noreply.github.com>
Co-authored-by: Doug Bunting <6431421+dougbu@users.noreply.github.com>
Co-authored-by: Justin Kotalik <jukotali@microsoft.com>
Co-authored-by: David Fowler <davidfowl@gmail.com>
Co-authored-by: Jeow Li Huan <huan086@users.noreply.github.com>
Co-authored-by: Andrew Stanton-Nurse <andrew@stanton-nurse.com>
Co-authored-by: Steve Sanderson <SteveSandersonMS@users.noreply.github.com>
Co-authored-by: Nate McMaster <natemcmaster@users.noreply.github.com>
Co-authored-by: Smit Patel <smitpatel@users.noreply.github.com>

dougbu committed:
Merge branch 'release/2.2' => 'master' (#10589)

- [automated]

natemcmaster committed:
Mark BindModel_ReturnsModel_IfAttemptedValueIsValid_FrenchThirtyTwoThousandPointOne as flaky (#10857)

natemcmaster committed:
Merge branch 'release/3.0-preview6'

natemcmaster committed:
Update branding to 2.1.12 (#10865)

Tratcher committed:
Mark ShutdownTestRun as Flaky (#10852)

natemcmaster committed:
Merge branch 'release/2.1' into release/2.2

natemcmaster committed:
Prepare 2.2.6

rynowak committed:
Add support for TypeConverter (#10730)

* Add support for TypeConverter

Fixes: #8493
Fixes: #9632
Fixes: #9339
Fixes: #8385
Fixes: 10077

This fix adds support for type converters as well as a few other minor
things we were missing from binding. The key thing about supporting
conversions is that we new can support arbitrary types with `@bind`.
This means you can use it with generics, which is something many users
have tried.

Along with type converters we get Guid and TimeSpan from the BCL. The
BCL also includes converters for types we're less interested in like
`short`.

* Use correct NumberStyles

* Fix culture

* Core check

dotnet-maestro[bot] committed:
[release/3.0-preview6] Update dependencies from  2 repositories (#10856)

* Update dependencies from https://github.com/aspnet/EntityFrameworkCore build 20190604.2

- Microsoft.EntityFrameworkCore.Tools - 3.0.0-preview6.19304.2
- Microsoft.EntityFrameworkCore.SqlServer - 3.0.0-preview6.19304.2
- dotnet-ef - 3.0.0-preview6.19304.2
- Microsoft.EntityFrameworkCore - 3.0.0-preview6.19304.2
- Microsoft.EntityFrameworkCore.InMemory - 3.0.0-preview6.19304.2
- Microsoft.EntityFrameworkCore.Relational - 3.0.0-preview6.19304.2
- Microsoft.EntityFrameworkCore.Sqlite - 3.0.0-preview6.19304.2

Dependency coherency updates

- Microsoft.AspNetCore.Analyzer.Testing - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.AspNetCore.BenchmarkRunner.Sources - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.ActivatorUtilities.Sources - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Caching.Abstractions - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Caching.Memory - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Caching.SqlServer - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Caching.StackExchangeRedis - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.CommandLineUtils.Sources - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.Abstractions - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.AzureKeyVault - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.Binder - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.CommandLine - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.EnvironmentVariables - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.FileExtensions - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.Ini - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.Json - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.KeyPerFile - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.UserSecrets - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.Xml - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.DependencyInjection.Abstractions - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.DependencyInjection - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.DiagnosticAdapter - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Diagnostics.HealthChecks - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.FileProviders.Abstractions - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.FileProviders.Composite - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.FileProviders.Embedded - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.FileProviders.Physical - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.FileSystemGlobbing - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.HashCodeCombiner.Sources - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Hosting.Abstractions - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Hosting - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.HostFactoryResolver.Sources - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Http - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Localization.Abstractions - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Localization - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.Abstractions - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.AzureAppServices - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.Configuration - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.Console - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.Debug - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.EventSource - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.EventLog - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.TraceSource - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.Testing - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.ObjectPool - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Options.ConfigurationExtensions - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Options.DataAnnotations - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Options - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.ParameterDefaultValue.Sources - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Primitives - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.TypeNameHelper.Sources - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.ValueStopwatch.Sources - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.WebEncoders - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Internal.Extensions.Refs - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.JSInterop - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Mono.WebAssembly.Interop - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.CSharp - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Win32.Registry - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.ComponentModel.Annotations - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Diagnostics.EventLog - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.IO.Pipelines - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Net.Http.WinHttpHandler - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Net.WebSockets.WebSocketProtocol - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Reflection.Metadata - 1.7.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Runtime.CompilerServices.Unsafe - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Security.Cryptography.Cng - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Security.Cryptography.Pkcs - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Security.Cryptography.Xml - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Security.Permissions - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Security.Principal.Windows - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.ServiceProcess.ServiceController - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Text.Encodings.Web - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Text.Json - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- System.Threading.Channels - 4.6.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.DependencyModel - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- NETStandard.Library.Ref - 2.1.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.Platforms - 3.0.0-preview6.19303.8 (parent: Microsoft.NETCore.App)
- Microsoft.NETCore.App - 3.0.0-preview6-27803-13 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Internal.AspNetCore.Analyzers - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.AspNetCore.Testing - 3.0.0-preview6.19304.2 (parent: Microsoft.EntityFrameworkCore)

* fix tests

* Update dependencies from https://github.com/aspnet/AspNetCore-Tooling build 20190604.1

- Microsoft.NET.Sdk.Razor - 3.0.0-preview6.19304.1
- Microsoft.CodeAnalysis.Razor - 3.0.0-preview6.19304.1
- Microsoft.AspNetCore.Razor.Language - 3.0.0-preview6.19304.1
- Microsoft.AspNetCore.Mvc.Razor.Extensions - 3.0.0-preview6.19304.1

* fix mvc test

* virtual test

Commit:
Update gRPC package versions (#10757)

dougbu committed:
Merge branch 'master' into merge/release/3.0-preview6-to-master

BrennanConroy committed:
[Preview 6] Use JsonReader interop with JsonSerializer (#10733)

Commit:
Update dependencies from https://github.com/aspnet/EntityFrameworkCore build 20190604.10

- Microsoft.EntityFrameworkCore.Tools - 3.0.0-preview6.19304.10
- Microsoft.EntityFrameworkCore.SqlServer - 3.0.0-preview6.19304.10
- dotnet-ef - 3.0.0-preview6.19304.10
- Microsoft.EntityFrameworkCore - 3.0.0-preview6.19304.10
- Microsoft.EntityFrameworkCore.InMemory - 3.0.0-preview6.19304.10
- Microsoft.EntityFrameworkCore.Relational - 3.0.0-preview6.19304.10
- Microsoft.EntityFrameworkCore.Sqlite - 3.0.0-preview6.19304.10

Dependency coherency updates

- Microsoft.AspNetCore.Analyzer.Testing - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.AspNetCore.BenchmarkRunner.Sources - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.ActivatorUtilities.Sources - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Caching.Abstractions - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Caching.Memory - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Caching.SqlServer - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Caching.StackExchangeRedis - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.CommandLineUtils.Sources - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.Abstractions - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.AzureKeyVault - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.Binder - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.CommandLine - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.EnvironmentVariables - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.FileExtensions - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.Ini - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.Json - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.KeyPerFile - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.UserSecrets - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration.Xml - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Configuration - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.DependencyInjection.Abstractions - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.DependencyInjection - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.DiagnosticAdapter - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Diagnostics.HealthChecks - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.FileProviders.Abstractions - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.FileProviders.Composite - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.FileProviders.Embedded - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.FileProviders.Physical - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.FileSystemGlobbing - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.HashCodeCombiner.Sources - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Hosting.Abstractions - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Hosting - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.HostFactoryResolver.Sources - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Http - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Localization.Abstractions - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Localization - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.Abstractions - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.AzureAppServices - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.Configuration - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.Console - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.Debug - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.EventSource - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.EventLog - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.TraceSource - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Logging.Testing - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.ObjectPool - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Options.ConfigurationExtensions - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Options.DataAnnotations - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Options - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.ParameterDefaultValue.Sources - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.Primitives - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.TypeNameHelper.Sources - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.ValueStopwatch.Sources - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.WebEncoders - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Internal.Extensions.Refs - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.JSInterop - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Mono.WebAssembly.Interop - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.Extensions.DependencyModel - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- NETStandard.Library.Ref - 2.1.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.NETCore.App - 3.0.0-preview6-27804-01 (parent: Microsoft.Extensions.Logging)
- Microsoft.Extensions.Logging - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Internal.AspNetCore.Analyzers - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)
- Microsoft.AspNetCore.Testing - 3.0.0-preview6.19304.6 (parent: Microsoft.EntityFrameworkCore)

mmitche committed:
Merge pull request #10879 from aspnet/darc-release/3.0-preview6-e8223ad7-e33a-41a8-bcf8-0b8519d22d19

[release/3.0-preview6] Update dependencies from aspnet/EntityFrameworkCore

jkotalik committed:
Start using runtime properties to set StartupHook and EntrypointFilter (#10121)

natemcmaster committed:
Merge branch 'release/3.0-preview6'

natemcmaster committed:
Merge branch 'release/2.2'

BrennanConroy committed:
Merge branch 'release/3.0-preview6' into brecon/mergePreview6

MichalStrehovsky committed:
Remove use of GetRuntimeInterfaceMap in IsTypeAwaitable (#10572)

`GetRuntimeInterfaceMap` is a rather expensive API and it doesn't look like the use here is warranted.

The replacement should have equivalent behavior.

pranavkm committed:
Guarantee ordering for SignalR client HttpConnection.send (#10780)

* Revert "Revert "Add an option to SignalR to guarantee ordering of send events… (#10807)"
  - This reverts commit 39a9d63c10e25ec0fea81fb47e77c2cabbb2b5db.
* Fixup build

ryanbrandenburg committed:
Merge pull request #10841 from isaac2004/10346-work

Remove Cookie Consent UI from all templates and move into docs #10346

ryanbrandenburg commented:
@iantoalms it looks like there was a communication problem between AZDO and GitHub. The easiest way to fix it is probably to rebase your commit on to the latest master. Once we've got a passing build there we can merge this.

isaacrlevin committed:
Merge branch 'master' into master

Commit:
Disable OSX 10.14 tests on Helix and use OSX 10.13 instead (#10824)

BrennanConroy committed:
Merge pull request #10906 from aspnet/brecon/mergePreview6

Merge preview6 into master

saluce65 committed:
Remove gender-specific terminology from comments

saluce65 committed:
Update logout.component.ts

natemcmaster committed:
[automated] Merge branch 'release/2.2' => 'master' (#10905)

mikaelm12 committed:
Propagate cancellation token IAsyncEnumerators (#10901)

ryanbrandenburg committed:
Merge pull request #10808 from isaac2004/master

Simplify web project templates #10347

ajcvickers committed:
Cleanup in Identity tests

Fixes #10671 by using updated EF API
Fixes #5812 so tests are re-enabled for PRs
Also part of #10668 since one of the changed EF queries is now working

Finally, I removed all remaining platform skips since with SQLite these tests should now run anywhere.

davidfowl committed:
Added basic event counters for hosting (#10884)

- Added Request Rate counter (RPS), Total Requests counter, current requests and failed requests.
- Counters do nothing until enabled
- Added tests for counters and improved tests to use unique event source names
- Renamed EventSource to match the convention in the runtime Microsoft.AspNetCore.Hosting

ryanbrandenburg committed:
Merge pull request #10918 from saluce65/patch-2

Update logout.component.ts

ryanbrandenburg committed:
Merge pull request #10917 from saluce65/patch-1

Remove gender-specific terminology from comments

pranavkm committed:
Revert "Guarantee ordering for SignalR client HttpConnection.send (#10780)" (#10916)

This reverts commit b1a29281ba3b93d55a6b7b5b1a0c7f27de04bb78.

dotnet-maestro[bot] committed:
[master] Update dependencies from  2 repositories (#10863)

* Update dependencies

natemcmaster committed:
[automated] Merge branch 'release/3.0-preview6' => 'master' (#10910)

[automated] Merge branch 'release/3.0-preview6' => 'master'

jkotalik committed:
Add better error message when someone does single file publish in ANCM (#10871)

jkotalik committed:
Guarantee that ANCM package dlls are used for back compat tests (#10858)

scalablecory committed:
Fix a write past the end of buffer when IntegerEncoder encodes to multi-bytes and does not have sufficient room to do so. (#10914)

Add some basic asserts to constrain parameters to HPACK spec.
Add some additional tests.

BrennanConroy committed:
Add missing log to hub method failure (#10903)

halter73 committed:
Improve shutdown of SignalR .NET client sample (#10675)

analogrelay committed:
fix #7548 by updating to v142 toolset (again) (#10904)

* fix #7548 by updating to v142 toolset (again)

* small note

* it's a compiler bug. let's just try

* you get a pass vc++... THIS TIME

* should no longer need v141 toolset!

halter73 committed:
Unify C# and TS SignalR client auto-reconnect APIs (#10678)

BrennanConroy committed:
Change case-insensitive Json default in SignalR (#10814)

jkotalik committed:
Add versioning to dotnet-dev-certs (#10908)

iantoalms committed:
Merge branch 'FixForIssue10762' of github.com:iantoalms/AspNetCore into FixForIssue10762
